docs/dg/dev/architecture/coding-conventions.md

```diff
@@ -0,0 +1,2165 @@
+- **Guideline**: While not mandatory, following these guidelines is highly recommended. Doing so promotes long-term code maintainability and facilitates smoother development processes.
+
+# Applications
+Spryker utilises application layers into applications to enable constructing the necessary application architecture for the specific business requirements to provide a quick project start, and long-term maintainability.
+- Backend applications (eg: Zed, Backend API, Backend Gateway, Backoffice, GlueBackend, MerchantPortal) typically use the Zed-, Glue-, Client-, Service-, and Shared application layers.
+- Storefront applications (eg: Yves, Configurator, Glue, GlueStorefront) typically use the Yves-, Glue-, Client-, Service- and Shared application layers.
+
+# Application Layers
+Spryker organises responsibilities/functionalities over a set of application layers (see [Conceptual Overview](https://docs.spryker.com/docs/dg/dev/architecture/conceptual-overview.html)) to enable flexible business logic orchestration across the applications.
+
+The application layers are aggregations of layers (see [Modules and Application Layers](https://docs.spryker.com/docs/dg/dev/architecture/modules-and-application-layers.html))
+- **Zed**: is a **multi-layer composition** of the 4 layers: `Presentation Layer`, `Communication Layer`, `Business Layer`, `Persistence Layer`.
+- **Yves**: is a **single-layered** `Presentation Layer` merged with `Communication Layer`.
+- **Glue**: is a **single-layered** `Communication Layer`.
+- **Client**: is a **single-layered** `Communication Layer`.
+- **Service**: is a **single-layered**, overarching `Business Layer`.
+- **Shared**: is a layer-overarching, stateless, abstraction library.
+
+## Zed
+
+`Zed` serves as the backend application layer responsible for housing all business logic and persisting data.
+```
+[Organization]
+└── Zed
+    └── [Module]
+        ├── Presentation
+        │   ├── Layout
+        │   │   └── [layout-name].twig
+        │   └── [name-of-controller]
+        │       └── [name-of-action].twig
+        │    
+        ├── Communication
+        │   ├── Controller
+        │   │   ├── IndexController.php
+        │   │   ├── GatewayController.php
+        │   │   └── [Name]Controller.php
+        │   ├── [CustomDirectory]
+        │   │   ├── [ModelName]Interface.php
+        │   │   └── [ModelName].php        
+        │   ├── Exception
+        │   ├── Plugin
+        │   │   └── [ConsumerModule]
+        │   │       └── [PluginName]Plugin.php
+        │   ├── Form
+        │   │   ├── [Name]Form.php
+        │   │   └── DataProvider
+        │   │       └── [Name]FormDataProvider.php
+        │   ├── Table
+        │   │   └── [Name]Table.php                
+        │   ├── navigation.xml       
+        │   └── [Module]CommunicationFactory.php
```

Comment:
these folders do not fit the description above 

>`Zed` serves as the backend application layer responsible for housing all business logic and persisting data.

So update the description and include UI for Backoffice.